client/src/app/site/assignments/assignment-list/assignment-list.component.html

```diff
@@ -31,14 +32,23 @@
         <mat-header-cell *matHeaderCellDef mat-sort-header>Title</mat-header-cell>
         <mat-cell *matCellDef="let assignment">{{ assignment.getTitle() }}</mat-cell>
     </ng-container>
+    <!-- pahse column-->
     <ng-container matColumnDef="phase">
         <mat-header-cell *matHeaderCellDef mat-sort-header>Phase</mat-header-cell>
         <mat-cell *matCellDef="let assignment">
             <mat-chip-list>
                 <mat-chip color="primary" selected>{{ assignment.phase }}</mat-chip>
             </mat-chip-list>
-        </mat-cell>
+        </mat-cell><button mat-menu-item (click)="toggleMultiSelect()">
```

Comment:
looks falsely aligned. Did that run through prettier? Usually prettier does not accept `</something><something>`